tracks/ruby/exercises/clock/GUIDE.md

```diff
@@ -0,0 +1,249 @@
+# Clock
+
+Clock introduces students to the concept of **value objects** and **modular
+arithmetic**.
+
+## Reasonable solutions
+
+```ruby
+class Clock
+  MINUTES_PER_HOUR = 60
+  MINUTES_PER_DAY = 1440
+
+  def initialize(hour: 0, minute: 0)
+    @total_minutes = ((hour * MINUTES_PER_HOUR) + minute) % MINUTES_PER_DAY
+  end
+
+  def to_s
+    hours = total_minutes / MINUTES_PER_HOUR
+    minutes = total_minutes % MINUTES_PER_HOUR
```

Comment:
Would you pull this two variables out to their own methods?

Reply:
I like it 👍